site/src/common/blocks/RichTextBlock.tsx

```diff
@@ -27,61 +28,60 @@ const defaultRenderers: Renderers = {
      */
     blocks: {
         // Paragraph
-        unstyled: (children, { keys }) => children.map((child, idx) => <Text key={keys[idx]}>{child}</Text>),
+        unstyled: (children, { keys }) =>
+            children.map((child, idx) => (
+                <StyledTyphography component={"p"} key={keys[idx]} gutterBottom>
```

Comment:
I think it would be cleaner to remove the `{}` from all the `variant` and `component` props. 

```suggestion
                <StyledTyphography component="p" key={keys[idx]} gutterBottom>
```

Reply:
There would be a linter rule, to automatically change / force that. 
I would suggest to add the linter rule to use it consistent project wide, but that would be an own PR.
@thomasdax98 should we add that rule?
Is that what we are looking for [jsx-curly-brace-presence](https://github.com/jsx-eslint/eslint-plugin-react/blob/master/docs/rules/jsx-curly-brace-presence.md)

Reply:
@johnnyomair how can this be a breaking change? This rule should only apply to the code style in this repo and has no effect on interfaces? Or do i miss something?